tests/integration/components/identity-steps/step-two-test.js

```diff
@@ -26,7 +26,15 @@ module('Integration | Component | identity-steps/step-two', function (hooks) {
       );
     assert
       .dom('[data-test-getting-started-paragraph-2]')
-      .hasText('https://github.com/identity-service/instructions.md');
+      .hasText(
+        'After exploring the sample profile service, proceed to the next step by clicking the button below.',
+      );
+    assert
+      .dom('.profile-service__link')
```

Comment:
please use data-test for selecting an element

Reply:
done